compiler/rustc_parse/src/errors.rs

```diff
@@ -451,6 +451,15 @@ pub(crate) struct MissingExpressionInForLoop {
     pub span: Span,
 }
 
+#[derive(Diagnostic)]
+#[diag(parse_loop_else)]
+pub(crate) struct LoopElseNotSupported {
+    #[primary_span]
+    #[suggestion(applicability = "machine-applicable", code = "")]
```

Comment:
I don't think this is a useful suggestion and it's not machine-applicable at all since it's probably not what the user intended.

Reproducing the exact behavior from python in Rust isn't possible generally with a , you need to set a boolean variable in front of each break if you have them. So I don't think it's possible to give a structured suggestion here.

Maybe a `note` saying that the user should use a boolean variable to control whether the block is run instead would be enough?

Reply:
Thanks, I wasn't sure about the suggestion either. How is this `note`?
> consider moving this `else` clause to a separate `if` statement and use a `bool` variable to control if it should run

Reply:
sounds good, might be a little on the long side but I think it's fine